tests/Runner/FileLintingIteratorTest.php

```diff
@@ -37,7 +37,6 @@ public function testFileLintingIteratorEmpty(): void
             $fileLintingIteratorProphecy->reveal()
         );
 
-        static::assertNull($fileLintingIterator->current());
```

Comment:
I'm wondering, why this change?

Reply:
PHPStan reported that the assertion cannot pass and I removed it too fast. I restored it, I think the error is actually a false positive:
```
------ -------------------------------------------------------------------------------------------------------------- 
  Line   tests/Runner/FileLintingIteratorTest.php                                                                      
 ------ -------------------------------------------------------------------------------------------------------------- 
  40     Call to static method PHPUnit\Framework\Assert::assertNull() with SplFileInfo will always evaluate to false.  
 ------ --------------------------------------------------------------------------------------------------------------
```